Dockerfile

```diff
@@ -1,14 +1,25 @@
-FROM python:3.10-slim
+FROM python:3.10-slim as base
+FROM base as builder
+
+RUN pip install -U pip wheel \
+    && mkdir /app /wheelhouse
+
+COPY requirements.txt /app/
+COPY src /app/src
+
+RUN cd /app/ && pip wheel -r requirements.txt --wheel-dir=/wheelhouse
+
+FROM base
 
 LABEL maintainer="Plone Community <dev@plone.org>"  \
       org.label-schema.name="code-quality" \
       org.label-schema.description="Plone code quality tool" \
       org.label-schema.vendor="Plone Foundation" \
       org.label-schema.docker.cmd="docker run -rm -v "${PWD}":/github/workspace plone/code-quality check black src"
```

Comment:
With the new configuration it is better to remove the `src` at the end of the command.

Reply:
No need, as this builder image is "thrown away" after building the final image.